README.md

```diff
@@ -1,3 +1,39 @@
-# coSMicQC
+<img height="200" src="https://raw.githubusercontent.com/WayScience/coSMicQC/main/logo/with-text-for-light-bg.png?raw=true">
 
-coSMicQC is a software tool for single-cell quality control of morphology datasets.
+# Single cell Morphology Quality Control
```

Comment:
Would it make sense for the abbreviation / project name to also be mentioned here in a parenthetical?
```suggestion
# Single cell Morphology Quality Control (coSMicQC)
```

Reply:
Hmm, I am going to not add this, I think in the logo is includes this abbreviation so to me it feels a bit repetitive. We can change this later though!